.env.test

```diff
@@ -4,6 +4,6 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
-TRELLO_API_KEY=4effc87f8e3d5da536c3f923d87aacd6
-TRELLO_API_TOKEN=fe1682e1d7089060fdbd4321a87c27b598eb381c348707a7b23a345c5405af31
-TRELLO_BOARD_ID=620d0d2b67ba96254c06c246
\ No newline at end of file
+TRELLO_API_KEY=
+TRELLO_API_TOKEN=
```

Comment:
You should have some values here, just not the real ones.

E.g. if you have `TRELLO_API_KEY=test-key` then when you run tests you could check whether your app makes requests to Trello with `?key=test-key`